src/kitovu/cli.py

```diff
@@ -99,6 +99,13 @@ def edit(config: typing.Optional[pathlib.Path] = None, editor: typing.Optional[s
 
     if config is None:
         config = settings.get_config_file_path()
+        if not config.exists():
+            if not config.parent.exists():
```

Comment:
Both `.mkdir()` and `.touch()` have an `exists_ok=True` parameter (it's even `True` by default for `.touch()`), so you don't need these checks with that.

Reply:
Ah, `exists_ok=True` makes more sense.

For example vim is not able to create the file if the folder `~/.conf/kitovu` doesn't exist. So we could only create the folder, but I guess there are editors which require the file to exist (I think of not-so-smart Windows editors :wink:)